control_data_collecting_tool/README.md

```diff
@@ -23,125 +50,175 @@ This package provides tools for automatically collecting data using pure pursuit
 
 2. Set an initial pose, see [here](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/planning-simulation/#2-set-an-initial-pose-for-the-ego-vehicle).
 
-3. Add `DataCollectingAreaSelectionTool` rviz plugin.
+3. Add the DataCollectingAreaSelectionTool and DataCollectingGoalPlugin RViz plugins by clicking the "+" icon at the top of the RViz window.
 
-   <img src="resource/add_rviz_plugin.png" width="480">
+<img src="resource/add_rviz_plugin.png" width="480">
 
-4. Launch control_data_collecting_tool.
+4.  Launch control_data_collecting_tool.
 
-   ```bash
-   ros2 launch control_data_collecting_tool control_data_collecting_tool.launch.py
-   ```
+    ```bash
+    ros2 launch control_data_collecting_tool control_data_collecting_tool.launch.py map_path:=:=$HOME/autoware_map/sample-map-planning
```

Comment:
```suggestion
    ros2 launch control_data_collecting_tool control_data_collecting_tool.launch.py map_path:=$HOME/autoware_map/sample-map-planning
```

Reply:
I apologize for the delayed response. I have made the necessary corrections in 3537a91189a4a22c0b0ac7eec0f42ebd361eb40a.